docs/vs-tools-tizen/packaging/how-to-customize-tpk.md

```diff
@@ -1,418 +1,434 @@
-# How to customize TPK package
+# Customizing TPK Packages
+
+You can customize TPK packages in various ways.
+
+For more information, see:
```

Comment:
These links do not work.

Reply:
You had better remove "For more information, see:" section.

Reply:
Removed